Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -8,6 +8,29 @@ import GetFeatureInfoFormat from "./GetFeatureInfoFormat.js";
 import TimeDynamicImagery from "./TimeDynamicImagery.js";
 import UrlTemplateImageryProvider from "./UrlTemplateImageryProvider.js";
 
+/**
+ * EPSG codes known to include reverse axis orders, but are not within 4000-5000.
+ *
+ * @type {number[]}
+ */
+var includesReverseAxis = [
+  3034, // ETRS89-extended / LCC Europe
+  3035, // ETRS89-extended / LAEA Europe
+  3042, // ETRS89 / UTM zone 30N (N-E)
+  3043, // ETRS89 / UTM zone 31N (N-E)
+  3044, // ETRS89 / UTM zone 32N (N-E)
+];
+
+/**
+ * EPSG codes known to not include reverse axis orders, and are within 4000-5000.
+ *
+ * @type {number[]}
+ */
+var excludesReverseAxis = [
+  4471, // Mayotte
+  4559, // French Antilles
+];
```

Comment:
Out of curiosity how did you come up with this list and do you have a sense of how exhaustive it is? It should be possible to get a full list with a proj script. Doesn't necessarily need to be a blocker for the PR though since it's a good incremental improvement.

Reply:
https://github.com/mapserver/mapserver/issues/3582
I got the current list from this mapserver issue and used epsg.io to confirm each of them.
I've never used proj4 before, if you can provide me with some more info on how I could get the full list then I will give it a shot.